tests/Storage.test.ts

```diff
@@ -15,7 +15,7 @@ import * as assert from 'assert';
 import { LedgerStorage } from '../src/modules/storage/LedgerStorage';
 import { Enrollment } from '../src/modules/data';
 
-var sample_data =
+let sample_data =
```

Comment:
```suggestion
const sample_data =
```
If you change it, const seems suitable.

Reply:
Okay

Reply:
I fixed it, thank you for review.